src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -2,9 +2,7 @@
 
 import java.io.FileNotFoundException;
 import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
+import java.nio.file.*;
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation here. Need to specify all the imported packages instead of using `*`.